pkg/api/schema.graphql

```diff
@@ -0,0 +1,83 @@
+schema {
+    query: Query
+}
+
+# Leave date as a string (specify YYYY-MM-DD) and convert later?
+# How do we do latest/earliest?
+#scalar Date
+
+# Query contains the entry points into the schema
+type Query {
+    #bug(id: Int!, datestamp: String = "latest"): Bug
+    # TODO: Add sorting argument for bugs? asc/desc + id/assigned_to/component/etc/etc...default to ???
+    bugs(datestamp: String = "_latest", components: [String!]): [Bug]!
+    snapshot(datestamp: String = "_latest"): Snapshot
+    # Should this just be "graphdata"? All lowercase vs camelcase?
```

Comment:
Should this apply to just the entry point name?  Or also to all the variables (ie, targetRelease) too?  Does graphql have a preference, or do we stick with go camelcase?